### ISSUE TITLE
Support multiple test points, interpolate between results

### ISSUE THREAD

**Issue by tinyfluffs** [Deutherius/VGB#10]
Assuming I am doing a cold test, followed by tests at regular heating intervals, and finally a hot test - the script crashes. If I do a single cold/test/hot, I am able to generate the results.

Is there not a way to interpolate the mesh from a given set of data as it's heating up? Seems like it would help drastically.

**Comment by Deutherius** [comment]
Yes, the script only expected one TEST mesh to be present in the original printer.cfg file. I have pushed an update, it should now be possible to have multiple TEST meshes and the script will print out an accuracy test for each one.

If I understand your question correctly, you are also asking if it would be possible to calculate VGBs from multiple meshes that are taken during heating instead of just COLD and HOT? I assume it would be possible, but I am not sure if it would be worthwhile - the thermal expansion coefficient for each bed mesh point seems fairly linear and the interpolation "good enough" with errors on the order of units of microns. 

If you feel like you do need non-linear compensation, there is [FDC](https://github.com/JiMcNulty/FDC), which supersedes VGB in many aspects.
